go.mod

```diff
@@ -354,10 +353,10 @@ require (
 )
 
 replace (
-	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.25.0-sdk-v0.46.16
+	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.25.0-sdk-v0.46.16.0.20241009080849-0cd36203b354
```

Comment:
[note for reviewers]
will be updated once we merge the PRs

Reply:
what is the status of those PRs? any timeline on when you are going to merge them and make a release?

Reply:
yes, as soon as the prototype is validated, and this PR is ready to merge. I am leaving them open so that we do changes on them if the changes on node require any without having to make multiple releases

Reply:
ok. that makes sense. thank you. So once we agree on some solution here, then you'll merge those PRs and make a release.

Reply:
yes